source/demo/ContentBox.css

```diff
@@ -1,5 +1,5 @@
 .ContentBox {
-  flex: 1;
+  flex: 1 0 auto;
```

Comment:
This is obviously not directly related to the source changes, but something to this effect was required to make my addition to the demo work (a ContentBox was shrinking and scrolling, rather than `.Body`). I spot checked the other demo changes and didn't see anything obviously broken. Removing `overflow: auto` has the same effect, if that is preferable to you.